### PR TITLE
Fix/minor bugs

### PR DESCRIPTION
**Pull request by AlefrankM**

### CODE REVIEW

**Review by america-nishi — APPROVED** [review]
looks good

**Review by america-nishi — APPROVED** [review]

File: src/features/channels/components/channel-list/channel-list-item/ChannelListItem.tsx [hunk 1]
```diff
@@ -25,7 +26,7 @@ type Props = {
 const ChannelListItem = ({ channel, avatar = channel.channelName, selected = false }: Props) => {
```
Comment (felygarcia) [inline src/features/channels/components/channel-list/channel-list-item/ChannelListItem.tsx:26]: Fix this. unused constant

File: src/features/channels/components/channel-list/channel-list-item/ChannelListItem.tsx [hunk 2]
```diff
@@ -108,7 +111,22 @@ const ChannelListItem = ({ channel, avatar = channel.channelName, selected = fal
           secondary={
             <Box display="flex">
               <Typography noWrap component="span" variant="caption" color="text.secondary">
-                {channel.messages.length ? channel.messages[0].decryptedMessage : ''}
+                {!!channel.messages.length && (
+                  <>
+                    {channel.messages[0].messageType === 'attachment' && (
+                      <div
+                        style={{
```
Comment (felygarcia) [inline src/features/channels/components/channel-list/channel-list-item/ChannelListItem.tsx:118]: Styles should not be in the component, fix it

File: package.json [hunk 3]
```diff
@@ -96,7 +102,7 @@
     "prettier": "^2.7.1",
     "socket.io": "4.5.1",
     "ts-node": "^10.9.1",
-    "typescript": "4.7.4",
+    "typescript": "*",
```
Comment (felygarcia) [inline package.json:105]: It is safer to specify a version where it is known to work

File: src/features/channels/components/channel-list/channel-list-item/ChannelListItem.tsx (hunk 2 above)
Comment (AlefrankM) [reply src/features/channels/components/channel-list/channel-list-item/ChannelListItem.tsx:118]: Go

**Review by america-nishi — APPROVED** [review]
looks good to my eyes

**Pull request merged**